### DISCUSSION TITLE
Export of Database Results in CSV format

### DISCUSSION THREAD

**Discussion by inhouari** [metgem/metgem#113] [General]
Hello, 

I have some difficulties to export my database results in MetGem. 
In fact, when I try to do it i think a step is missing in front of the case "Fied Separator". (see the picture I joint)
I also try to go into "?" to help me but it doesn't work either. Can you please explain why it does not work and what can I do to succeed it ? 

Thanks a lot,

inhouari

<img width="223" alt="issues_metgem" src="https://user-images.githubusercontent.com/129958168/233391923-1f2d0e84-cbd3-4e4d-a580-d3520070b538.png">

**Comment by n-elie** [comment]
Hello inhouari,

Thanks for reporting. It seems to be a bug in the validation of the field separator.
Could you please give me the version of MetGem you are using? I can't reproduce this in the latest development version.
You could also try to choose "Other" in the dropdown list and type a field separator like `,` in the text box.

**Comment by inhouari** [comment]
Hello, 
Thank you for your answer. 

I use 1.3.6 version of MetGem. 
I try all the field seperator but the same bug appeared. 

I also tried to put ',' in the text box with 'Other', the same bug does not appeared. However, I can see another error and my file is not created. (see th picture of the bug message). Details of the bug : 

Traceback (most recent call last):
  File "metgem_app\ui\main_window.py", line 1139, in export_db_results
  File "metgem_app\ui\export_db_results_dialog.py", line 121, in getValues
AttributeError: 'QLineEdit' object has no attribute 'delimiter'


Thank you again, 

inhouari

<img width="246" alt="bug_metgem" src="https://user-images.githubusercontent.com/129958168/234055873-798b1f20-f7b6-4017-a05b-6f9dbd899fce.png">

**Comment by n-elie** [comment]
Please update to a development version (1.4.1). You can find it here: https://github.com/metgem/metgem/releases/tag/nightly
This should solve your problem

**Reply by inhouari** [reply]
n-elie,

Thanks for the help. It works perfectly. 

inhouari
